scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -28,35 +28,49 @@ include::modules/ztp-du-host-networking-requirements.adoc[leveloffset=+1]
 
 include::modules/ztp-acm-preparing-to-install-disconnected-acm.adoc[leveloffset=+1]
 
-[id="installing-preparing-mirror_{context}"]
-== Preparing the mirror host
-
-Before you perform the mirror registry procedure, you must prepare the mirror host to retrieve the {product-title} release images.
-
-include::modules/installation-about-mirror-registry.adoc[leveloffset=+2]
-
-.Additional information
-
-For information on viewing the CRI-O logs to view the image source, see xref:../installing/validating-an-installation.html#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
 
 [IMPORTANT]
 ====
 You must have access to the internet to obtain the necessary container images.
 In this procedure, you place the mirror registry on a mirror host
-that has access to both your network and the internet. If you do not have access
+that has access to both your network and the Internet. If you do not have access
 to a mirror host, use the disconnected procedure to copy images to a device you
 can move across network boundaries with.
 ====
 
-include::modules/installation-creating-mirror-registry.adoc[leveloffset=+2]
+[id="prerequisites_installing-mirroring-installation-images_{context}"]
+=== Prerequisites
+
+* You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2] in the location that will host the {product-title} cluster, such as one of the following registries:
++
+--
+** link:https://www.redhat.com/en/technologies/cloud-computing/quay[Red Hat Quay]
+** link:https://jfrog.com/artifactory/[JFrog Artifactory]
+** link:https://www.sonatype.com/products/repository-oss?topnav=true[Sonatype Nexus Repository]
+** link:https://goharbor.io/[Harbor]
+--
+If you have an entitlement to Red Hat Quay, see the documentation on deploying Red Hat Quay link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/[for proof-of-concept purposes] or link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html/deploy_red_hat_quay_on_openshift_with_the_quay_operator/[by using the Quay Operator]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat support.
+
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+2]
+
+.Additional information
```

Comment:
We typically use "Additional resources" for nested includes and anchors (see [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies)).

```suggestion
.Additional resources
```